### ISSUE TITLE
Add station ID (RPID) to the GTS IODA files

### ISSUE THREAD

**Issue by jiaruidong2017** [JCSDA-internal/ioda-converters#628]
## Description
In the bufr_snow_adpsfc.yaml, there are not entry yet to read in the station identifier (RPID) from the adpsfc dump file. We will require the station ID for the duplication check and our later QCs (e.g., in the rejectlist filter and acceptlist filter). 

The station IDs are in string formats such as "ABL", "CNCO", "03023", "Maale Gi". 

I tried to include the RPID in the yaml file, but I just got the strange values as "0", "Infinityf" and "1e+11". 

## Requirements

>If this is a new feature: What does the new code need to accomplish? Does it require new software dependencies (e.g. new jedi-stack components or new python modules?)

Station IDs will be included in the ioda output files.

>If this is a bugfix: What is the expected behavior?

## Acceptance Criteria (Definition of Done)
>What does it mean for this to be finished?

Station IDs are added in the IODA output files so that several QCs including the rejectlist filter and acceptlist filter might be applied.

## Dependencies
>What must be done before this can be done? Add issue dependencies in ZenHub as appropriate
>Does this block progress on other issues? Add this issue as a dependency to other ZenHub issues as appropriate

**Comment by jiaruidong2017** [comment]
I just made a test that the new version bufr2ioda.x could extract station ID from adpsfc dump file successfully. I will make more tests, and make sure the extracted IDs are correct. Thanks.

**Comment by BenjaminRuston** [comment]
@jiaruidong2017 since you made successful test should we close and open new issue as needed

**Comment by jiaruidong2017** [comment]
Thanks Ben for reminder.
